tests/Responses/Expected/checkout/reselect_shipping_response.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": @integer@,
+    "items": @array@,
```

Comment:
The problem is, it's ``applyStateMachineTransitionAction`` from ``ResourceController``, so we don't have many options on manipulating the response.